shared/reports/editable.py

```diff
@@ -14,15 +14,31 @@ class EditableReportFile(ReportFile):
 
     __slots__ = ("_details",)
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    @classmethod
+    def from_ReportFile(cls, report_file: ReportFile):
+        name = report_file.name
+        editable_file = cls(name)
+        editable_file._totals = report_file._totals
+        editable_file._session_totals = report_file._session_totals
+        editable_file._lines = report_file._lines
+        editable_file._line_modifier = report_file._line_modifier
+        editable_file._ignore = report_file._ignore
+        editable_file._details = report_file._details
+        editable_file.fix_details()
```

Comment:
This already happened in the `__init__` method when calling `cls(name)` above.  Do we need to do it again here?

Reply:
Oh I think the answer is "yes" since we're setting `_details` right above this line.

Reply:
Exactly, I think we do need to do that again.